openshift-knative-operator/pkg/monitoring/monitoring_eventing_test.go

```diff
@@ -19,8 +19,15 @@ func TestLoadPlatformEventingMonitoringManifests(t *testing.T) {
 		t.Errorf("Got %d, want %d", len(manifests), 1)
 	}
 	resources := manifests[0].Resources()
-	if len(resources) != 28 {
-		t.Errorf("Got %d, want %d", len(resources), 28)
+
+	// We create a service monitor and a service monitor service per deployment.
```

Comment:
that (SM and Service for SM) is the `*2` - right?

Reply:
Yes exactly